src/python/pyhbac.c

```diff
@@ -20,13 +20,14 @@
 
 #include "config.h"
 
-#include <Python.h>
-#include <structmember.h>
-
 #include "util/util.h"
 #include "util/sss_python.h"
 #include "lib/ipa_hbac/ipa_hbac.h"
 
+// Python headers are included last because they enforce _POSIX_C_SOURCE
```

Comment:
Please:
(1) use '/* */'-style comments (https://sssd.io/contrib/coding-style.html#general-rules)
(2) provide more context (something like as in https://pagure.io/SSSD/sssd/issue/4027 : "_POSIX_C_SOURCE=200809  turns off Annex K extensions that defines errno_t..." )

And also please add "Resolves: https://github.com/SSSD/sssd/issues/4997" to the commit message.

Reply:
Btw, did you check if this compiles on FreeBSD?

Reply:
Yes.